Dockerfile

```diff
@@ -14,16 +18,17 @@ RUN apk add --update bash \
   && LDFLAGS="-X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
   && cd .. \
   && export GOPATH=/gopath \
-  && REPO_PATH="github.com/Financial-Times/people-rw-neo4j" \
-  && mkdir -p $GOPATH/src/${REPO_PATH} \
-  && cp -r people-rw-neo4j/* $GOPATH/src/${REPO_PATH} \
+  && REPO_ROOT="github.com/Financial-Times/" \
+  && REPO_PATH="$REPO_ROOT/${PROJECT}" \
+  && mkdir -p $GOPATH/src/${REPO_ROOT} \
+  && mv ${PROJECT} $GOPATH/src/${REPO_ROOT} \
   && cd $GOPATH/src/${REPO_PATH} \
   && go get ./... \
   && cd $GOPATH/src/${REPO_PATH} \
   && echo ${LDFLAGS} \
   && go build -ldflags="${LDFLAGS}" \
-  && mv people-rw-neo4j /app \
+  && mv ${PROJECT} / \
   && apk del go git \
   && rm -rf $GOPATH /var/cache/apk/*
 
-CMD [ "/app" ]
\ No newline at end of file
+CMD [ "/people-rw-neo4j" ]
```

Comment:
I wanted to but people suggested that it would be a bit too obtuse. Also, would have to pass it as an ENV which in theory would mean you could override it when you ran the container. Not sure if that would be a good or bad thing :/